src/Microsoft.Bcl.HashCode/src/Microsoft.Bcl.HashCode.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+      <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>

```

Comment:
Are these the correct target frameworks to use?

Reply:
.NET Framework 4.6.1 is already out-of-support. Use the TFM properties defined in Arcade: https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/TargetFrameworkDefaults.props

Reply:
Do I need to use `$(NetFrameworkMinimum);$(NetFrameworkCurrent)` or just `$(NetFrameworkCurrent)`?

Reply:
For .NET Framework, it's fine to just target the minimum supported version: `$(NetFrameworkMinimum)`. Take a look at the System.Buffers project in the repo.

Reply:
Judging by the comment in that arcade file, I might need to use `netstandard2.0;$(NetFrameworkMinimum)`